src/fluent-plugin-mdsd/gemspec-template

```diff
@@ -0,0 +1,30 @@
+Gem::Specification.new do |s|
+    s.name = "fluent-plugin-mdsd"
+    s.version = "GEMVERSION"
+    s.date = "2017-01-05"
+    s.summary = "Fluentd output plugin for Linux monitoring agent (mdsd)"
+    s.description = "Fluentd output plugin for Linux monitoring agent(mdsd), which sends data to Azure Geneva service."
+    s.authors = ["azlinux"]
+    s.email = "azlinux@microsoft.com"
```

Comment:
>azlinux@microsoft.com [](start = 15, length = 21)

I am pretty sure that alias is not externally addressable. That is, I don't think you can send email to azlinux@microsoft.com from outside the company. We may have to setup a new alias for this. Check with Anand.